packages/tasks_panel/src/commands/edit-task.ts

```diff
@@ -23,12 +23,6 @@ export async function editTreeItemTask(
 }
 
 async function editTask(task: ConfiguredTask, readResource: (file: string) => Promise<string>): Promise<void> {
```

Comment:
edit task report is not interesting to report?

Reply:
there is a "task save" event that covers this